pkg/controller/pagerdutyintegration/clusterdeployment_created.go

```diff
@@ -93,20 +93,56 @@ func (r *ReconcilePagerDutyIntegration) handleCreate(pdclient pd.Client, pdi *pa
 	// To prevent scoping issues in the err check below.
 	var pdIntegrationKey string
 
+	// load configuration
 	err = pdData.ParseClusterConfig(r.client, cd.Namespace, configMapName)
```

Comment:
Shouldn't we still be doing something with this error?

Reply:
checking `err` now in the if that follows